### DISCUSSION TITLE
Firmware update via OTA removes uploading

### DISCUSSION THREAD

**Discussion by e-i-k-e** [tbnobody/OpenDTU#613] [General]
Hello,
I can't get the "opendtu-generic.bin" uploaded from the finished zip file. It only goes up to 6-14% and then doesn't change anymore.

A restart or another browser brought no improvement.

![image](https://user-images.githubusercontent.com/83473829/220187406-c62f8ff3-d5b1-4649-b263-78eee5179809.png)

![image](https://user-images.githubusercontent.com/83473829/220187928-0c863193-efa6-4abc-aa27-a49f0cfbbac2.png)

**Comment by tbnobody** [comment]
Have you already tried a complete power cycle? (Not just restart in the web Interface)

**Reply by e-i-k-e** [reply]
There isn't. ;)
Thanks!
